Tests/ClientTests/TelemetryWrapperTests.swift

```diff
@@ -154,16 +154,6 @@ class TelemetryWrapperTests: XCTestCase {
                                   failureMessage: "Should have \(expectTabCount) tabs for private tabs")
     }
 
-    func test_tabsNormalQuantityWithoutExtras_GleanIsNotCalled() {
-        TelemetryWrapper.recordEvent(category: .information, method: .background, object: .tabNormalQuantity, value: nil, extras: nil)
-        XCTAssertNil(GleanMetrics.Tabs.normalTabsQuantity.testGetValue())
-    }
-
-    func test_tabsPrivateQuantityWithoutExtras_GleanIsNotCalled() {
-        TelemetryWrapper.recordEvent(category: .information, method: .background, object: .tabPrivateQuantity, value: nil, extras: nil)
-        XCTAssertNil(GleanMetrics.Tabs.privateTabsQuantity.testGetValue())
-    }
-
```

Comment:
We don't need these at all now? Even after the incoming patch?

Reply:
I removed them and will add them back when the bug fix with Glean happens. We don't have a clean store, so getting nil for tabs will not be possible unless we force a specific order on the tests. There's existing tests through TabsQuantityTelemetryTests for `normalTabsQuantity` and `privateTabsQuantity` in the meantime.